src/app/+item-page/field-components/metadata-field-wrapper/metadata-field-wrapper.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { MetadataFieldWrapperComponent } from './metadata-field-wrapper.componen
 @Component({
     selector: 'ds-component-with-content',
     template: '<ds-metadata-field-wrapper [label]="\'test label\'">\n' +
-      '    <div class="my content">\n' +
+      '    <div class="my-content">\n' +
```

Comment:
If the problem/bug is caused by empty `<span>`s, should we add an empty `<span>` into this test data to just prove that the `textContent` check is working accurately?